### PR TITLE
Add support of ME in --owners

### PR DESCRIPTION
**Pull request by romasku**
Closes https://github.com/neuro-inc/platform-client-python/issues/1834

### CODE REVIEW

File: CLI.md [hunk 1]
```diff
@@ -511,7 +511,7 @@ Name | Description|
 |_--format COLUMNS_|Output table format, see "neuro help ps\-format" for more info about the format specification. The default can be changed using the job.ps-format configuration variable documented in "neuro help user-config"|
 |_\--full-uri_|Output full image URI.|
 |_\-n, --name NAME_|Filter out jobs by name.|
-|_\-o, --owner TEXT_|Filter out jobs by owner \(multiple option).|
+|_\-o, --owner TEXT_|Filter out jobs by owner \(multiple option). Supports `ME` option to filter by current users username.|
```
Comment (asvetlov) [inline CLI.md:514]:
```suggestion
|_\-o, --owner TEXT_|Filter out jobs by owner \(multiple option). Supports `ME` option to filter by the current user.|
```

File: neuromation/cli/job.py [hunk 2]
```diff
@@ -555,7 +555,8 @@ async def attach(root: Root, job: str, port_forward: List[Tuple[int, int]]) -> N
     "-o",
     "--owner",
     multiple=True,
-    help="Filter out jobs by owner (multiple option).",
+    help="Filter out jobs by owner (multiple option). "
+    "Supports `ME` option to filter by current users username.",
```
Comment (asvetlov) [inline neuromation/cli/job.py:559]:
```suggestion
    "Supports `ME` option to filter by the current user.",
```

**Review by asvetlov — APPROVED** [review]
Looks good, please apply the small message correction

**Comment by serhiy-storchaka** [comment]
If support `ME` as an alias of the current user name in `--owners`, should not we also replace `<you>` with `ME` in the `neuro ps` output?

**Review by asvetlov — APPROVED** [review]

**Review by serhiy-storchaka — APPROVED** [review]

**Review by asvetlov — APPROVED** [review]

**Pull request merged**